util/topsql/reporter/client.go

```diff
@@ -79,16 +154,28 @@ func (r *GRPCReportClient) Send(ctx context.Context, targetRPCAddr string, data
 	}()
 	wg.Wait()
 	close(errCh)
-	for err := range errCh {
+	for err = range errCh {
 		if err != nil {
-			return err
+			return
 		}
 	}
-	return nil
+
+	return
+}
+
+// IsPending implements ReportClient interface.
+func (r *GRPCReportClient) IsPending() bool {
+	return len(config.GetGlobalConfig().TopSQL.ReceiverAddress) == 0
```

Comment:
Will `TopSQL.ReceiverAddress` have races when it is modified?

Reply:
`config.GetGlobalConfig()` ensures that `TopSQL.ReceiverAddress` is immutable.